.github/workflows/build_release.yaml

```
@@ -0,0 +1,66 @@
+name: Rust Build and Release
+
+on:
+  push:
+    branches: [ main ]
+    tags:
+      - 'v*'
```

```suggestion
    tags:
      - 'v*'
```

These conditions are `OR` so even if you specify `branches: [ main ]`, you can't limit the Git tags to be pushed from any branches 🙏 Therefore, it's better to remove it!

Ref: [Trigger Github Action only on new tags?, Stackoverflow](https://stackoverflow.com/questions/61891328/trigger-github-action-only-on-new-tags)